src/qseek/exporters/velest.py

```diff
@@ -2,34 +2,275 @@
+                phase_s = phase
+
+        catalog = search.catalog
+
+        # export station file
+        stations = search.stations.stations
+        station_file = outdir / "stations_velest.sta"
+        self.export_station(stations=stations, filename=station_file)
+
+        # export phase file
+        phase_file = outdir / "phase_velest.pha"
+        n_earthquakes = 0
+        for event in catalog:
+            if event.semblance < min_event_semblance:
+                continue
+            if event.receivers.n_observations(phase_p) < min_receivers_number:
+                continue
+            if event.distance_border < min_distance_to_border:
+                continue
+
+            observed_arrivals: list[tuple[Receiver, PhaseDetection]] = []
+
+            for receiver in event.receivers:
+                for _phase, detection in receiver.phase_arrivals.items():
+                    if detection.observed is None:
+                        continue
+                    observed = detection.observed
+                    if (
+                        detection.phase == phase_p
+                        and observed.detection_value <= min_p_phase_confidence
+                    ):
+                        continue
+                    if (
+                        detection.phase == phase_s
+                        and observed.detection_value <= min_s_phase_confidence
+                    ):
+                        continue
+                    if (
+                        detection.traveltime_delay.total_seconds()
+                        > max_traveltime_delay
+                    ):
+                        continue
+                    observed_arrivals.append((receiver, detection))
+
+            countp, counts = self.export_phases_slim(
+                phase_file, event, observed_arrivals
+            )
+            self.n_picks_p += countp
+            self.n_picks_s += counts
+            n_earthquakes += 1
+        self.n_events = n_earthquakes
+
+        # export control file
+        control_file = outdir / "velest.cmn"
+        control_file_parameters = VelestControlFile(
+            ref_lat=search.octree.location.lat,
+            ref_lon=-search.octree.location.lon,
+            n_earthquakes=n_earthquakes,
+        )
+        control_file_parameters.write_config_file(control_file)
+        # export velocity model file
+        dep = search.ray_tracers.root[0].earthmodel.layered_model.profile("z")
+        vp = search.ray_tracers.root[0].earthmodel.layered_model.profile("vp")
+        vs = search.ray_tracers.root[0].earthmodel.layered_model.profile("vs")
+        dep_velest = []
+        vp_velest = []
+        vs_velest = []
+        km = 1000.0
+        for d, vpi, vsi in zip(dep, vp, vs, strict=False):
+            if float(d) / km not in dep_velest:
+                dep_velest.append(float(d) / km)
+                vp_velest.append(float(vpi) / km)
+                vs_velest.append(float(vsi) / km)
```

Comment:
Hi, thank you so much for the review! This is because the velocity model for velest requires only one value for each depth, but the velocity model of qseek is layered model with two velocity at one depth, so I need to remove another value.